<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimePlotWindow</class>
 <widget class="QMainWindow" name="RealTimePlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomPlot" name="accPlot" native="true"/>
    </item>
    <item>
     <widget class="QCustomPlot" name="gyroPlot" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="terminalButton">
        <property name="text">
         <string>terminal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveGraphButton">
        <property name="text">
         <string>save graph</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="instantValuesButton">
        <property name="text">
         <string>instantaneous values</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>RealTimePlotWindow</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
